.travis.yml

```diff
@@ -2,7 +2,7 @@ matrix:
   include:
     - language: java
       script:
-        - make validate-canonical-match
+        - 'if [[ "$TRAVIS_PULL_REQUEST" = "false" && "master" = "$TRAVIS_BRANCH" ]]; then make deploy-license-data; else make validate-canonical-match; fi'
```

Comment:
I don't think we need the wrapping single-quote here.  For a similar example that does not use it, see [here][1].

[1]: https://github.com/spdx/spdx-spec/blob/1f7f2a0213eceba89aec010b89f286fc6e0a0a21/.travis.yml#L18